testing-guidance/workflow-tests/o3de-project-manager/gems/gem-workflow-tests.md

```diff
@@ -136,3 +136,27 @@ See [sample-code-gems repo.json](https://github.com/o3de/sample-code-gems/blob/m
 | Add **Remote Gem** to an **O3DE Project**.               | <ol><li>Add **Remote Gem(s)** to the **Gem Remote Sources Manager.**</li><li>Configure your project to add your **Remote Gem(s)**.</li><li>Build and open your project in the O3DE Editor.</li></ol>                                                                                                                                                                                                                    | <ul><li>**Remote Gem(s)** can be added & removed from an **O3DE Project**.</li><li>Known good **Remote Gem(s)** should build.</li><li>**Remote Gem(s)** content can be interacted with from the O3DE Editor. <ul><li>EG: Assets are processed and available in asset manager if assets are present.</li><li>EG: Editor Entity Components can be added if Editor Entity Components are present.</li><li>EG: Script Canvas nodes can be added to a graph if Script Canvas nodes are present in the gem.</li></ul></li></ul>|
 ---
 
+
+### Workflow: Verify Gem's version and engine compatibility in Project Manager
+
+**Workflow Docs**
+
+[O3DE Gem Repositories](https://www.o3de.org/docs/user-guide/gems/repositories/overview/)
+
+**Workflow Requirements**
+* O3DE built
+* Gem with at least two versions
```

Comment:
Do we have a link to a gem repo that satisfies this requirement? If so we should probably include it as a suggestion if the tester doesn't already have one.

Reply:
Unfortunately currently I do not have the access to the mentioned gem repository.